### ISSUE TITLE
Update Documentation from IDS to DSP

### ISSUE THREAD

**Issue by mhellmeier** [eclipse-tractusx/tractusx-edc#537]
The [migration guide from 0.3.4 to 0.4.0](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/migration/Version_0.3.4_0.4.0.md) mentions the switch from the IDS protocol to DSP. Nevertheless, different parts of the documentation (`docs` folder) still use the deprecated IDS protocol, for example, the [Transfer Data sample](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/docs/samples/Transfer%20Data.md).

**Comment by ndr-brt** [comment]
thanks for the issue, not a bug though

**Comment by mhellmeier** [comment]
The issue is still relevant, documentation is still outdated.

**Comment by mhellmeier** [comment]
This issue is still relevant; information about the DSP is still missing.

**Comment by ndr-brt** [comment]
the documentation changed quite a lot with #932 , where a lot of "IDS" references had been removed, could you point what is missing and what needs to be changes?

**Comment by mhellmeier** [comment]
> [...], could you point what is missing and what needs to be changed?

Of course! For me, general information about the data space protocol (DSP) and its usage with its implementation is missing.

Further, different parts are still outdated. All endpoints changed from `/api/v1/ids` to `/api/v1/dsp` (based on the [migration guide](https://github.com/eclipse-tractusx/tractusx-edc/blob/07c2bcf1a0713082cd1fb382645c14aedd259a12/docs/migration/Version_0.3.4_0.4.0.md)). Nevertheless, the old endpoints are still used in the documentation. Examples:
https://github.com/eclipse-tractusx/tractusx-edc/blob/07c2bcf1a0713082cd1fb382645c14aedd259a12/docs/kit/Operation%20View/02_Local%20Setup/00_local_setup_controlplane.md?plain=1#L55
https://github.com/eclipse-tractusx/tractusx-edc/blob/07c2bcf1a0713082cd1fb382645c14aedd259a12/docs/kit/Operation%20View/02_Local%20Setup/00_local_setup_controlplane.md?plain=1#L59

**Comment by mhellmeier** [comment]
Maybe related to #953 because it makes pretty good steps towards clearer documentation, but the Operation View is still outdated. What would be the final steps? Move everything to https://github.com/eclipse-tractusx/eclipse-tractusx.github.io to prevent duplicate content? Completely remove the operation view?

**Comment by ndr-brt** [comment]
there's a pr going on for the rework of the connector kit: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/558
once that's merged some further cleanup can be done here

**Comment by mhellmeier** [comment]
Okay, so let's wait until the mentioned PRs are merged. Afterward, we can check how we can go on here.

**Comment by arnoweiss** [comment]
@mhellmeier - can you check if this is solved now?

**Comment by mhellmeier** [comment]
> @mhellmeier - can you check if this is solved now?

No, it isn't solved and is still relevant. There are outdated things as [described above](https://github.com/eclipse-tractusx/tractusx-edc/issues/537#issuecomment-1880804790).

**Comment by arnoweiss** [comment]
There's two problems discussed here

### Explaining the DSP
I agree with @ndr-brt - this should be done in the upstream eclipse-edc. Additionally, the DSP docs are linked everywhere in the Management API Walkthrough.

### Outdated docs scattered across tractusx-edc
Especially for deployment config. Traces of "ids" are a good signal for outdated docs. Having docs for config-properties in this repo makes sense because this repo should be a one-stop-shop for building, deploying and using the tractusx-edc. This doesn't exist anywhere else:
- The MXD explains [deployment of several connectors](https://github.com/eclipse-tractusx/tutorial-resources/blob/main/mxd/README.md) with terraform, but not on the property-level.
- [eclipse-edc docs](https://github.com/eclipse-edc/Connector/tree/main/docs/developer) are aimed at developers, not operations folk.

That's why I'm suggesting:
1. Remove the Operation View from tractusx-edc
2. Remove all examples containing config properties with `*.ids.*`. Those are mainly located in the [edc-controlplane](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-controlplane) and [edc-dataplane](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/edc-dataplane) folders.
3. Exclusively point to the documentation in the charts folder. There, a [full list of properties](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/charts/tractusx-connector-azure-vault/README.md#values) is auto-generated.
4. Remove all "Known issues" sections as known issues are likely handled as github-issues.
